v2/cli/init_command.go

```diff
@@ -0,0 +1,248 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+type InitFields struct {
+	ID       string
+	Language string
+	Git      string
+}
+
+var initCommand = &cli.Command{
+	Name:  "init",
+	Usage: "Initialize a template Brigade project",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    flagID,
+			Aliases: []string{"i"},
+			Usage: "Name of the Brigade project to create " +
```

Comment:
I would suggest changing this to "Name of the Brigade project to initialize". I believe that will improve clarity on what this command does. As it's currently worded, a user could pretty easily be fooled into thinking that `brig project create` takes place automatically when they do `brig init`

The notes will eventually add further clarity, but this will help.